ui/imports/shared/popups/keycard/KeycardPopup.qml

```diff
@@ -485,6 +487,7 @@ StatusModal {
         },
         StatusButton {
             id: primaryButton
+            objectName: "PrimaryButton"
```

Comment:
nit if could be more precise about what this is, just in case some one else add primary button, that will create conflict